packages/esm-forms-dashboard-app/src/forms-dashboard/forms-dashboard.component.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { getConfig } from '@openmrs/esm-framework';
+import { useTranslation } from 'react-i18next';
+import styles from './forms-dashboard.scss';
+import { EmptyDataIllustration, useVisitOrOfflineVisit } from '@openmrs/esm-patient-common-lib';
+import FormsList from './form-list/forms-list.component';
+import { useFormsToDisplay } from '../hooks/use-forms';
+import { DataTableSkeleton, Layer, Tile } from '@carbon/react';
+import { HtmlFormEntryForm } from '@openmrs/esm-patient-forms-app/src/config-schema';
```

Comment:
This is actually exactly the kind of thing we want to avoid as we now have a strong dependency between `esm-froms-dashboard-app` and `esm-patient-forms-app`, i.e., they are no longer self-contained. We should either just duplicate the interface here or define it in, e.g. `@openmrs/esm-patient-common-lib` (which is intended to be shared).

Reply:
We have discussed this specif case in one of our meetings.
In this case I am using the `getConfig` to use the `esm-patient-forms-app` config so we don't have to replicate the configuration to open the O2 forms. 
So I think it make sence also use the interfaces for that config that already exist.